### ISSUE TITLE
building error

### ISSUE THREAD

**Issue by tinastefaniewenzel** [btubbs/heroku-buildpack-shiny#9]
Hi, 

I got an error when trying to deploy an app: 

`/app/vendor/R/lib64/R/bin/exec/R: error while loading shared libraries: libreadline.so.5: cannot open shared object file: No such file or directory`

I deployed as follows: 
`heroku apps:create APPNAME
        heroku config:set BUILDPACK_URL=https://github.com/btubbs/heroku-buildpack-shiny.git
        git push heroku
`
(The web socket command as described in the read.me is no longer needed - it returns an error in any case)

I then get the error: 
`/app/vendor/R/lib64/R/bin/exec/R: error while loading shared libraries: libreadline.so.5: cannot open shared object file: No such file or directory
`

Below is what's loaded in the environment on the heroku app

`
           BUILDPACK_URL=https://github.com/btubbs/heroku-buildpack-shiny.git 
           LDFLAGS=-L/app/vendor/gcc-4.3/lib64
          TERM=xterm-256color
          CPPFLAGS=-I/app/vendor/glibc-2.7/string -I/app/vendor/glibc-2.7/time -I/app/vendor/cairo/include/cairo/include
               LD_LIBRARY_PATH=/app/vendor/cairo:/app/vendor/fontconfig:/app/vendor/pixman:/app/vendor/libpng:/app/vendor/DirectFB:/app/vendor/freetype
            COLUMNS=99
            DYNO=run.5864
            PATH=/app/vendor/R/bin:bin:/app/vendor/gcc-4.3/bin:/usr/local/bin:/usr/bin:/bin
            PWD=/app
             R_INCLUDE=/app/vendor/R/lib64/R/include
            PS1=\[\033[01;34m\]\w\[\033[00m\] \[\033[01;32m\]$ \[\033[00m\]
           LINES=23
           HOME=/app
           SHLVL=2
             PKG_CONFIG_PATH=/app/vendor/cairo/lib/pkgconfig:/app/vendor/fontconfig/lib/pkgconfig:/app/vendor/pixman/lib/pkgconfig:/app/vendor/libpng/lib/pkgconfig:/app/vendor/DirectFB/lib/pkgconfig:/app/vendor/freetype/lib/pkgconfig
           R_HOME=/app/vendor/R
           PORT=53747
           _=/usr/bin/env
`

The full login output is:
`
            Starting process with command`R -e "shiny::runApp('.', as.numeric(Sys.getenv(\"PORT\")))"`
 WARNING: ignoring environment value of R_HOME
2016-09-30T12:00:25.267616+00:00 app[web.1]: /app/vendor/R/lib64/R/bin/exec/R: error while            loading shared libraries: libreadline.so.5: cannot open shared object file: No such file or directory
            State changed from starting to crashed 
            State changed from crashed to starting
            Process exited with status 127
            Starting process with command`R -e "shiny::runApp('.', as.numeric(Sys.getenv(\"PORT\")))"`
            WARNING: ignoring environment value of R_HOME
/app/vendor/R/lib64/R/bin/exec/R: error while loading shared libraries: libreadline.so.5: cannot open                  shared object file: No such file or directory
            Process exited with status 127
           State changed from starting to crashed
`

Any suggestions how to fix this ? 
Thanks !

**Comment by feydan** [comment]
Same issue 4 years late, any fixes?

**Comment by btubbs** [comment]
Nope.  If anyone wants to pitch in and maintain this buildpack, that would be great.
